jpa/README.md

```diff
@@ -244,17 +243,17 @@ Click on the terminal tab.
 At the terminal prompt access psql for your database
 
 ```shell
-sh-4.2$ psql sampledatabase
-psql (12.3)
+sh-4.2$ psql sampledb
+psql (9.6.10)
```

Comment:
The change from 12.3 => 9.6.10 seems a bit surprising?

Reply:
It is what my current test showed. Direct cut and paste. But i don't think it matters as that is just a sample output.